Makefile

```diff
@@ -417,8 +417,8 @@ catalog-push: ## Push a catalog image.
 
 .PHONY: container-build
 container-build: ## Build containers
```

Comment:
Adding a comment here explaining why catalog-build isn't being called here would be really helpful